src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  margin: 0 auto;
+  width: 100%;
+  background-color: $bg-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  z-index: 1000;
+
+  &__container {
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
+
+  &__logo-link {
+    display: block;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__item {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    padding: 0;
+    margin: 0;
+    font-weight: 500;
+    height: 60px;
+    font-size: 12px;
+    text-transform: uppercase;
+    font-family: $roboto-font;
+    color: $text-color;
+    line-height: $header-size;
```

Comment:
The `line-height: $header-size;` property in `.nav__link` class is a bit confusing. It would be better to set a fixed value for the line-height, like `1.5` or `2`, to ensure consistent spacing.